lib/core/src/animation/physics/inertia.rs

```diff
@@ -296,8 +296,18 @@ impl PhysicsSimulator {
             }
 
             let transition = interval_counter.accumulated_time / interval_counter.interval_duration;
-            let position   = linear_interpolation(current_position,next_position,transition as f32);
-            callback(position);
+            let t          = transition as f32;
```

Comment:
I know the next line would be split, but split is better than single char name :P Lets name it `transition` instead :)